services/static-webserver/client/source/class/osparc/desktop/paymentMethods/PaymentMethods.js

```diff
@@ -184,7 +184,7 @@ qx.Class.define("osparc.desktop.paymentMethods.PaymentMethods", {
         spacing: 3,
         height: 150,
```

Comment:
Perhaps we could increase the height of the table. I don't know how many items we might get in here but I have 5 and I need to scroll. What do you think?

Reply:
Yes, I tried to quickly fix it but I could not, so I moved on. This is not trying to enhance anything but rather quickly fix some critical bugs